lib/workflow/postgres/postgres.go

```diff
@@ -153,6 +154,22 @@ func (pg *PostgresBackend) Init() error {
 	}
 	// defer m.Close() I don't think we want to call close here because it'll close our db instance
 
+	version, dirty, err := m.Version()
+	if err != nil && err != migrate.ErrNilVersion {
+		return errors.Wrap(err, "init migrator - error getting migration version")
+	}
+
+	if dirty {
```

Comment:
That's #597, isn't it?

Reply:
Yup, I'll rebase this once that merges.  I have it on this branch because it is not fun to iterate on schema changes without it.